manifests/role/mgetit_log.pp

```diff
@@ -12,4 +12,5 @@
   include nebula::profile::named_instances
   include nebula::profile::nodejs
   include nebula::profile::php73
+  include nebula::profile::ruby
```

Comment:
Before this is merged we need to be sure that the torquebox setup on malt is insulated from the change. This was a factor in the past when we tried to apply puppet-managed ruby to malt. I believe a change was made on malt to insulate TB, but we should verify.

Reply:
Good to know

Reply:
We're not messing with malt after all.